configs/base/packages.yaml

```diff
@@ -24,6 +24,8 @@ packages:
     variants: ~fortran
   trilinos:
     variants: ~adelus~adios2~amesos~amesos2~anasazi~asan+aztec~basker~belos~boost~chaco~complex~dtk~epetra~epetraext~epetraextbtf~epetraextexperimental~epetraextgraphreorderings~float~fortran~hypre~ifpack~ifpack2~intrepid~intrepid2~ipo~isorropia~mesquite~minitensor~ml~muelu~mumps~nox~openmp~panzer~phalanx~piro~python~rol~rythmos~sacado~scorec~shards~shylu~stk_simd~stk_unit_tests~stokhos~stratimikos~strumpack~suite-sparse~superlu~superlu-dist~teko~tempus~test~thyra~trilinoscouplings~x11~zoltan2
+    require:
+      - any_of: ["@=15.1.1", "@=develop"]
```

Comment:
I think we need to keep the `13.4.1` release as an option as well, but you can put it last so it is the lowest precedent.

If not then we won't be able to build the `exawind@1.0` release.